src/components/common/icons/ExportLink.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 
-const ExternalLink = ({ titleTag = 'External Link', width = 24, fill = '#ffffff' }) => (
+const ExternalLink = ({ titleTag = 'External Link', width = 24, fill = '#ffffff', margin }) => (
   <svg
     xmlns="http://www.w3.org/2000/svg"
     viewBox="0 0 24 20"
     style={{
-      width
```

Comment:
Rather than modifying the whole svg icon at source, best to style it contextually.
My aim with icons was to keep props simple and fairly predictable - i.e. they all have a width.
I suggest:
`const ExportButton = styled(IconButton)'svg {  margin-right: 1rem; }'`;

Sorry for the poor formatting, github can be a bit of a pain...

Reply:
I tried but there something overriding my styles so either I do some changes to the common component or create another one.
I will at them again during the weekend